docs/usage/managed_seed.md

```diff
@@ -0,0 +1,74 @@
+# Register Shoot as Seed
+
+An existing shoot can be registered as a seed by creating a `ManagedSeed` resource. This resource replaces the `use-as-seed` annotation that was previously used to create [shooted seeds](shooted_seed.md). It contains:
+
+* The name of the shoot that should be registered as seed.
+* An optional `seedTemplate` section that contains the `Seed` spec and parts of its metadata, such as labels and annotations.
+* An optional `gardenlet` section that contains:
+    * `gardenlet` deployment parameters, such as the number of replicas, the image, etc.
+    * The `GardenletConfiguration` resource that contains controllers configuration, feature gates, and a `seedConfig` section that contains the `Seed` spec and parts of its metadata.
+    * Additional configuration parameters, such as the garden connection bootstrap mechanism (bootstrap token, service account, or none), and whether to merge the provided configuration with the configuration of the parent `gardenlet`.
```

Comment:
You could refer to https://github.com/gardener/gardener/blob/master/docs/concepts/gardenlet.md#tls-bootstrapping here.

Reply:
Done